protocols/shared-workspace.md

```diff
@@ -0,0 +1,22 @@
+# Shared Workspace Protocol
+
+## Abstract
+Focusing in a shared workspace (or in the office in general) can be a really challenging task. Working from home solves the problem, but it is **not** an ideal solution if forced by the difficulty to focus in the office.
+
+The "Shared Workspace Protocol" ("SWP" henceforth) provides a simple set of rules for people working in the same workspace.
+
+## Language
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [BCP 14](https://tools.ietf.org/html/bcp14) \[[RFC2119](https://tools.ietf.org/html/rfc2119)\] \[[RFC8174](https://tools.ietf.org/html/rfc8174)\] when, and only when, they appear in all capitals, as shown here.
+
+## Goals
+SWP provides a reusable collaboration model for people working in the same workspace. It has these specific goals:
+
+1. To maximize focus and productivity, by reducing the sources of distraction.
+1. To reduce the burden to ask people to "behave" or "leave the room", by having an explicit protocol people agree before entering the room.
+
+## Design
+1. A person SHOULD write a message to another person before speaking with them.
+1. A person SHOULD NOT be loud.
+1. A person SHOULD NOT talk to more than one person at a time.
+1. One-on-one conversations MUST be short and occasional.
+1. A person MUST NOT have meetings or calls unless everyone in the room is involved.
```

Comment:
I should remove "meetings". Technically, I can have a meeting on IRC. /cc @ttmc

Reply:
Yeah, this rule is implied by the others, I think.